src/renate/benchmark/datasets/nlp_datasets.py

```diff
@@ -142,14 +142,13 @@ def tokenize_fn(batch):
 class MultiTextDataModule(DataIncrementalDataModule):
     """
     Inspired by the dataset used in "Episodic Memory in Lifelong Language Learning"
-    by d’Autume et al. this is a collection of five different datasets that we call domains:
+    by d’Autume et al. this is a collection of four different datasets that we call domains:
```

Comment:
Why the change?

Reply:
originally, it was a collection of 5 datasets. after amazon review was removed, it dropped to 4.